scripts/run.sh

```diff
@@ -75,12 +75,21 @@ function install() {
                 --email $EMAIL -d $DOMAIN --logs-dir /etc/letsencrypt/logs
         fi
     fi
+
+    echo -e -n "${CYAN}(!)${NC} Enter the database name for your Bitwarden instance (ex. vault): "
+    read DATABASE
+    echo ""
+
+    if [ "$DATABASE" == "" ]
+    then
+        DATABASE="vault"
+    fi
     
     pullSetup
     docker run -it --rm --name setup -v $OUTPUT_DIR:/bitwarden \
         --env-file $ENV_DIR/uid.env bitwarden/setup:$COREVERSION \
         dotnet Setup.dll -install 1 -domain $DOMAIN -letsencrypt $LETS_ENCRYPT -os $OS \
-        -corev $COREVERSION -webv $WEBVERSION
+        -corev $COREVERSION -webv $WEBVERSION -dbname $DATABASE
```

Comment:
Database names can have spaces in them in SQL Server, please ensure this command parameter is text qualified when being written out or passed to the docker run command.

Reply:
Added quotes around the variable in `run.sh` and `run.ps1`.  Both worked properly when tested.